components/navigation.css

```diff
@@ -0,0 +1,84 @@
+.header-bar {
+  align-items: center;
+  justify-content: space-around;
+  background-color: var(--white-color);
+  color: var(--cta-color);
+  width: 100%;
+  padding: 1rem;
+}
+
+.header-left {
+  width: 10rem;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.web-logo {
+  align-items: center;
+  color: var(--btn-primary-color);
+  gap: 0.5rem;
+  font-size: 2rem;
+}
+
+.web-logo-icon {
+  font-size: 2.5rem;
+}
+
+.header-mid {
+  background-color: var(--white-color);
+  color: var(--cta-color);
+  width: 20rem;
+  border: 0.1rem solid var(--cta-color);
+  border-radius: 0.4rem;
+  padding: 0.2rem;
+}
+
+.search-bar {
+  width: 100%;
+  border: none;
+}
+
+.header-right {
+  width: 22rem;
+  align-items: center;
+  justify-content: space-between;
+  gap: 2rem;
+  width: 30%;
```

Comment:
there is any purpose for using width in %. elsewhere you used rem unit for width only where width 100% required that time you use %.

Reply:
yes for making responsivness as it gets minimized according to page size